tests/maths/test_ols_sklearn_diabetes.py

```diff
@@ -78,6 +80,13 @@ def test_r_squared():
     assert output == approx(expected)
 
 
+def test_r_squared_no_intercept():
+    A, b, model = get_data_and_fit_statsmodel(False)
```

Comment:
Could we change the flag to `add_intercept`, and change the call on this line to `add_intercept=False`?

Otherwise its a bit unclear from the call site what the flag is.

Thanks!

Reply:
Agreed - "positive logic" is considered more readable